xAPI.md

```diff
@@ -507,6 +507,45 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 ## 7.0 xAPI Components
 
+This section explains and shows graphically how different pieces of xAPI can fit together.  
+
+##### Tracking Experiences Through Systems
+
+![Data Flow in xAPI](/xAPIDataFlow.jpg)
+
+Figure 1 shows the tracking of experiences through the data flow of a series of systems.  The cloud in the picture 
+represents a Learning Management System, or LMS.  An LMS, if present, would likely encapsulate everything within 
+the cloud.  In either case, a user will be dependent on a relationship between content and one or more Learning 
+Record Stores.  It is also possible a learner is dependent on an Activity Provider to deliver and/or authorize use 
+of content to them.  Once this triangle of authorization and access is complete, a registration exists for that 
+user in that content for that LRS.  LRSs, at request of the user, can also share data to other LRSs that may or 
+may not be a part of the LMS framework.  Web Services, again as a part of or independent of the LMS, can pull 
+data from one or more LRSs, send them through other Web Services or LRSs.
+
+##### Activity Data and Metadata
+
+![xAPI Activity Data and Metadata](/xAPIMetadata.jpg)
```

Comment:
There's always a balance between clarity and accuracy when it comes to diagrams. I find multiple simpler diagrams in place of one diagram to rule them all help. (And of course [I always follow that advice myself...](http://tincanapi.co.uk/diagram.pdf)).